specs/cypress/run-cypress-mobile.yaml

```diff
@@ -0,0 +1,15 @@
+name: Run Cypress Headless on Mobile
```

Comment:
I'm by no means an expert on cypress, but my understanding is this command is just setting the height and width to a certain size, regardless of whether it's a mobile device. It may be useful to name this workflow "Run Cypress Headless on Mobile Viewport" and the other one "Open Cypress in Mobile Viewport" to make that more clear

Reply:
Thanks for the feedback @alokedesai . Let me provide a bit more feedback in regards to Cypress. Essentially there are 2 different ways to "run" Cypress.

`open` essentially opens the cypress app and allows users to click on which tests they want to run, etc
`run` essentially runs the cypress app headless and tests every spec file.

It is a bit confusing but that is why we chose that specific verbiage in regards to open vs run.

Does this help?

Reply:
[982f6ee](https://github.com/warpdotdev/workflows/pull/46/commits/982f6ee25cba047335f933bee60dcabd937eac4c) makes this change